src/components/layer-manager/layer-item/layer-item.vue

```diff
@@ -52,6 +53,7 @@ function changeTime(dateStart?: string, dateEnd?: string) {
   <div class="lux-layer-manager-item relative">
     <div class="w-full flex flex-nowrap items-start gap-x-2">
       <button
+        v-if="!is3d"
         class="fa-solid fa-bars cursor-move mt-1"
         :class="draggableClassName"
         :title="txtDraggableLabel"
```

Comment:
When testing, I noticed, we still need to handle the `onClickInfo` for the layer metadata on the button below.